### DISCUSSION TITLE
如何在nuxt3中使用image组件的懒加载

### DISCUSSION THREAD

**Discussion by zhaohg1** [youzan/vant#13105] [General]
我是下载的依赖是@vant/nuxt,在文档中我没找到，然后通过gpt回答我写一个plugin插件，我按照gpt的写了，然后用了import {lazyload} from ’vant‘和import {lazyload} from @vant/nuxt‘，都报错，不知道怎么用了

**Comment by tolking** [comment]
[lazyload](https://github.com/vant-ui/vant-nuxt?tab=readme-ov-file#lazyload)
